lib/cmd/fhc/target.js

```diff
@@ -104,32 +97,6 @@ function target(argv, cb) {
         ini.set(keys.KEY_ID, targ[keys.KEY_ID], "user");
       }
 
-      var _gotVersion = function() {
-        ini.save(function(err) {
-          var message;
-          if (targ.user) {
-            message = " User: " + targ.user;
-          } else {
-            message = " API Key: " + targ[keys.KEY_ID];
-          }
-          target.message = i18n._("Successfully targeted: ") + targ.target + message;
-          return cb(err, targ);
-        });
-      };
-
-      ngui({_ : []}, function(err, isFh3) {
-        if (err) {
-          if (targ.fhversion) {
-            ini.set('fhversion', targ.fhversion);
-          } else {
-            target.message += i18n._('Your login has expired - please log in again!\n ') + login.usage;
-          }
-          return _gotVersion();
-        }
-        var version = (isFh3) ? 3 : 2;
-        ini.set('fhversion', version);
-        return _gotVersion();
-      });
```

Comment:
Hi @trepel, tks for your test. 

However, the fhvsersion was removed so no make sense keeps this lines of code. 
Look that the message and all still the same. 

```
Your branch is up-to-date with 'origin/FH-3618'.
Camilas-MBP:fh-fhc cmacedo$ ./bin/fhc.js target https://testing.zeta.feedhenry.com/
Successfully targeted https://testing.zeta.feedhenry.com/ - now, log in.
fhc login <username> <password>
You are prompted for username/password if none are provided.
Camilas-MBP:fh-fhc cmacedo$ 

```
We need understand where you are facing the issue because it shows to be another reason and not this lines.